internal/stats/stats_service.go

```diff
@@ -257,7 +264,11 @@ func (g runtimeGauges) updateNumCPU(bt *binTextStats) {
 }
 
 func (g runtimeGauges) updateUptime(bt *binTextStats) {
-	if uptime, err := host.Uptime(); err != nil {
+	if g.proc == nil {
+		// gopsutil.process is not registered, skip gauge
+		return
+	}
+	if uptime, err := g.proc.CreateTime(); err != nil {
```

Comment:
The uptime would be calculated as: `time.Now() - g.proc.CreateTime()`.

Reply:
Thanks for catching that. Copied time.Time to ms utility from Nearcache PR. This way we won't make duplication when they are merged.

https://github.com/hazelcast/hazelcast-go-client/pull/808/commits/94f37a2947d456aedaab0b20093381cb89c23057

Reply:
OK, but I've removed that stuff from NC, since we will upgrade to Go 1.17 that one is not necessary anymore.